### DISCUSSION TITLE
Missing old releases

### DISCUSSION THREAD

**Discussion by MLNW** [structurizr/cli#158] [General]
### Description

Since a few day releases prior to v1.34.0 are missing from GitHub. This prevents the installation of older versions of the CLI (e.g., when only Java 8 is available). Is there another place where we should be able to get them from?

### Steps to reproduce

1. Go to https://github.com/structurizr/cli/releases
2. See that there are only 3 releases (at the time of writing)

### Severity

Minor

### Priority

I have no budget and there's no rush, please fix this for free

**Comment by simonbrowndotje** [comment]
Yes, many of the previous releases have been removed, but the tags still exist (https://github.com/structurizr/cli/tags), so it should be straightforward to checkout a specific tag and build your own release. If your organisation has a paid support contract, let me know and we can look at doing this for you.

**Reply by MLNW** [reply]
> Yes, many of the previous releases have been removed

Almost sounds like a concious decision. If so, what's the reason, if I may ask?

> checkout a specific tag and build your own release

That's what I ended up doing. It increases CI time but it's not dramatic.
